packages/flutter/lib/src/painting/box_decoration.dart

```diff
@@ -83,9 +83,8 @@ class BoxDecoration extends Decoration {
     this.backgroundBlendMode,
     this.shape = BoxShape.rectangle,
   }) : assert(shape != null),
-       // TODO(mattcarroll): Use "backgroundBlendMode == null" when https://github.com/dart-lang/sdk/issues/34180 is in.
        assert(
-         identical(backgroundBlendMode, null) || color != null || gradient != null,
+         backgroundBlendMode == null || color != null || gradient != null,
          'backgroundBlendMode applies to BoxDecoration\'s background color or '
          'gradient, but no color or gradient were provided.'
```

Comment:
while you're here, can you change were to was here? thanks

Reply:
Sure, done!